docs/variant-specific/special-fives.mdx

```diff
@@ -102,10 +102,16 @@ These conventions apply to the "Special-Fives" variants (e.g. "Pink-Fives", "Mud
 
 - _Pink Promise_ does not apply to rank clues that could represent a playable light-pink-five. (This is different from how _Pink Promise_ works in a pink-fives variant.)
 - Thus, when players receive a rank _Play Clue_, they have to account for the being both the true rank and any playable light-pink-fives.
+  - The consequence of this is that when the light-pink 4 is down, _Self Finesses_ will no longer work.
```

Comment:
This is about special 5s, not special 5s + light-pink. Please clarify that *any* 4 being down *might* prevent a self finesse where exactly occam's razor has priority over, say, another player blind playing.

Reply:
@newduke New commit should include the above changes.